web/book/src/reference/syntax/strings.md

```diff
@@ -15,7 +15,7 @@ derive {
 **Quoting and escape characters**
 
 To quote a string containing quote characters, use the "other" type of quote, or
-use the escape character ("\"), or use more quotes.
+use the escape character ("\\"), or use more quotes.
```

Comment:
```suggestion
use the escape character (`\`), or use more quotes.
```
Is that clearer? I worry that `"\\"` isn't the escape character?

Reply:
Nope - the markdown needs to escape the escape character - two `\` are required

Reply:
This works:

`````diff
diff --git a/web/book/src/reference/syntax/strings.md b/web/book/src/reference/syntax/strings.md
index ce103f80..8d0dd919 100644
--- a/web/book/src/reference/syntax/strings.md
+++ b/web/book/src/reference/syntax/strings.md
@@ -15,7 +15,7 @@ # Strings
 **Quoting and escape characters**
 
 To quote a string containing quote characters, use the "other" type of quote, or
-use the escape character ("\\"), or use more quotes.
+use the escape character (`\`), or use more quotes.
 
 ```prql
 from artists
`````